src/runner.js

```diff
@@ -54,6 +54,26 @@ export function run(stdout, stdin, stderr, argv) {
     stdin.fd
   );
 
+  const issues = configuration.validate();
+
+  if (issues.length > 0) {
```

Comment:
I think we can remove this `if` statement since `.map` and `.some` will still work on empty arrays.